src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java

```diff
@@ -270,6 +273,21 @@ public OperationCompletionRS updateTestItem(ReportPortalUser.ProjectDetails proj
 				.collect(toList());
 	}
 
+	private void checkInitialStatusAttribute(TestItem testItem) {
+		Optional<ItemAttribute> statusAttirbute = testItem.getAttributes()
+				.stream()
+				.filter(attribute -> attribute.getKey().equalsIgnoreCase(INITIAL_STATUS_ATTRIBUTE_KEY) && attribute.isSystem())
```

Comment:
possible npe if attribute key is null
CONSTANT.equals(variable)

Reply:
fixed